SWLOR.Game.Server/Service/CombatService.cs

```diff
@@ -470,8 +470,11 @@ public static AbilityResistanceResult CalculateAbilityResistance(NWCreature atta
             //Console.WriteLine("defenderTotal = " + defenderTotal);
             //Console.WriteLine("divisor = " + divisor);
 
-            result.DC = (int) (defenderTotal / divisor * 100);
-            result.Roll = RandomService.D100(1);
+            result.DC = (int)defenderTotal + 10;
```

Comment:
What numbers are coming out of this calculation with these changes? Looks like a 100 point scale dropping to a 20 point scale?

Reply:
The numbers are way more balanced in terms of players hitting each other. instead of all the calculations being done in the DC we split it up so the defender is on the DC and the attacker on the roll. It favors the attackers still but it will even out the wide variation in damages rolled because its not basing the scaled damage on a d100. But it also keeps if the attack is super focused in the force and the other user isn't the low chance of the defender hits correctly.